packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -20,6 +33,36 @@ def _ensure_routable():
     return _ensure_routable
 
 
+@retrying.retry(wait_fixed=2000,
+                stop_max_delay=120 * 1000,
+                retry_on_result=lambda ret: ret is False,
```

Comment:
There is no return False so this line does nothing; please remove it. Default is to exit once any return is registered